management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java

```diff
@@ -6,7 +6,7 @@
 import java.util.Map;
 
 @Controller
-class HomeController {
+public class HomeController {
```

Comment:
Just curiosity .... is there a reason why you adde `public` here ? did something fail?

Reply:
I was trying to make an instance of the HomeController to make some testing and it caused some errors maybe the test was not in the same package i will add the HomeController tests

Reply:
good, thanks